lib/invoca/utils/enumerable.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Invoca ::Enumerable extensions
+module Enumerable
+  def build_hash(res = {})
+    each do |x|
+      pair = block_given? ? yield(x) : x
+      res[pair.first] = pair.last if pair
+    end
+    res
+  end
+
+  class MultiSender
```

Comment:
I think this should also be a module in `Invoca::Utils`. No need to clutter `module Enumerable` with it.

Reply:
Moved. Would you test this independently? Or rely upon the Array tests?

Reply:
I like 1 test using Array, and 1 of our own class that includes Enumerable. That covers the 2 extremes.

Reply:
Done.